src/scitokens/scitokens.py

```diff
@@ -461,8 +462,7 @@ def _validate_jti(self, value):
         JTI, or json token id, should always pass.  It's mostly used for logging
         and auditing.
         """
-        # Fix for unused argument
-        del value
+        logging.info("Validating SciToken with jti: {0}".format(value))
```

Comment:
Please use a specific logging target for SciTokens instead of the root logger.  For example:

```
logger = logging.getLogger('scitokens')
...
logger.info("Validating SciToken...")
```

Reply:
Fixed.  But the full logging configuration is coming in a pull request in a bit that include the "configuration" file parsing.